src/commands/init.ts

```diff
@@ -0,0 +1,86 @@
+import { Input } from "@oclif/parser";
+import * as fs from "fs-extra";
+import * as path from "path";
+import cli from "cli-ux";
+
+import TartCommand from "../TartCommand";
+
+export default class Init extends TartCommand {
+  static description = "Initializes tart for this project";
+
+  static flags = {
+    ...TartCommand.flags,
+  };
+
+  async init() {
+    // Skip
+  }
+
+  async run() {
+    const { flags } = this.parse(
+      this.constructor as Input<typeof TartCommand.flags>
+    );
+
+    const configPath = flags.config as unknown as string;
+
+    const configPathExists = await fs.pathExists(configPath);
+
+    if (!configPathExists) {
```

Comment:
I think we should output some message if we detect a config file already if we're not going to give them the option to re-create the config.

Reply:
done!